x-pack/docs/en/rest-api/security/enroll-kibana.asciidoc

```diff
@@ -17,6 +19,10 @@ Enables a {kib} instance to configure itself for communication with a secured {e
 Use this API to enable a {kib} instance to configure itself for communications
 with an {es} cluster that already has security features enabled.
 
+When you first start {es}, the installation process generates an enrollment token for
```

Comment:
I don't know if this is relevant to put here. Who's the intended audience ? If it's end users, are they interested in how Kibana uses this internal only API ?